### ISSUE TITLE
Optimize "Enter Badge Details"

### ISSUE THREAD

**Issue by Annicelia** [mint-o-badges/badgr-ui#176]
For me as a user it is not clear why I should add or not add the hexagon. Can we add some wording like in the wireframes to mak eit clear?
<img width="734" alt="Bildschirmfoto 2024-04-08 um 17 15 20" src="https://github.com/mint-o-badges/badgr-ui/assets/164371483/5f49ad66-54c1-4569-ab7b-c017c1c24b09">
MacOS Browser Firefox

**Comment by TobiSchumacher** [comment]
Yes, I had a similar thought, also because we only have a hexagon frame when creating participation badges. May be, we could also just replace "hexagon" with "Use OEB layout" or something along those lines
